app/models/event.rb

```diff
@@ -227,6 +227,19 @@ def days
     (end_date.to_date - start_date.to_date).to_i + 1
   end
 
+  def schedule_counts
```

Comment:
Let's move this to the EventStats class as that's where we'd like to keep all stat related logic.  It might even combine with the schedule_day_stats method.